definitions/npm/openai_v4.x.x/flow_v0.142.x-/openai_v4.x.x.js

```diff
@@ -0,0 +1,2515 @@
+     * @param {number} [opts.httpAgent] - An HTTP agent used to manage HTTP(s) connections.
+     * @param {Fetch} [opts.fetch] - Specify a custom `fetch` function implementation.
+     * @param {number} [opts.maxRetries] - The maximum number of times the client will retry a request.
+     * @param {Headers} opts.defaultHeaders - Default headers to include with every request to the
+     * @param {DefaultQuery} opts.defaultQuery - Default query parameters to include with every request to the
+     * @param {boolean} [opts.dangerouslyAllowBrowser] - By default, client-side use of this library is not allowed, as it risks exposing your secret API credentials to attackers.
+     * @param {string | null} [opts.organization]
+     */
+    constructor(x?: ClientOptions): this;
+    completions: Completions;
+    chat: Chat;
+    edits: Edits;
+    embeddings: Embeddings;
+    files: Files;
+    images: Images;
+    audio: Audio;
+    moderations: Moderations;
+    models: Models;
+    fineTuning: FineTuning;
+    fineTunes: FineTunes;
+    defaultQuery(): DefaultQuery | void;
+    defaultHeaders(opts: FinalRequestOptions<>): Headers;
+    authHeaders(opts: FinalRequestOptions<>): Headers;
+    static OpenAI: typeof OpenAI;
+    static OpenAIError: typeof Errors$OpenAIError;
+    static APIError: typeof Errors$APIError;
+    static APIConnectionError: typeof Errors$APIConnectionError;
+    static APIConnectionTimeoutError: typeof Errors$APIConnectionTimeoutError;
+    static APIUserAbortError: typeof Errors$APIUserAbortError;
+    static NotFoundError: typeof Errors$NotFoundError;
+    static ConflictError: typeof Errors$ConflictError;
+    static RateLimitError: typeof Errors$RateLimitError;
+    static BadRequestError: typeof Errors$BadRequestError;
+    static AuthenticationError: typeof Errors$AuthenticationError;
+    static InternalServerError: typeof Errors$InternalServerError;
+    static PermissionDeniedError: typeof Errors$PermissionDeniedError;
+    static UnprocessableEntityError: typeof Errors$UnprocessableEntityError;
+  }
+  declare var OpenAIError: typeof Errors$OpenAIError;
+  declare var APIError: typeof Errors$APIError;
+  declare var APIConnectionError: typeof Errors$APIConnectionError;
+  declare var APIConnectionTimeoutError: typeof Errors$APIConnectionTimeoutError;
+  declare var APIUserAbortError: typeof Errors$APIUserAbortError;
+  declare var NotFoundError: typeof Errors$NotFoundError;
+  declare var ConflictError: typeof Errors$ConflictError;
+  declare var RateLimitError: typeof Errors$RateLimitError;
+  declare var BadRequestError: typeof Errors$BadRequestError;
+  declare var AuthenticationError: typeof Errors$AuthenticationError;
+  declare var InternalServerError: typeof Errors$InternalServerError;
+  declare var PermissionDeniedError: typeof Errors$PermissionDeniedError;
+  declare var UnprocessableEntityError: typeof Errors$UnprocessableEntityError;
```

Comment:
Do we need all of these? They're not exported or used anywhere

Reply:
I'm not sure what they're for, so I removed them in ea5121c3.